### ISSUE TITLE
Data Quality

### ISSUE THREAD

**Issue by betterapp** [BlackbitDigitalCommerce/pimcore-data-director#88]
Can we do this on PimCore 6.9 ? I tried and looks like it is not working.

https://www.youtube.com/watch?v=M1VVmpTjB_I

**Comment by BlackbitDevs** [comment]
You mean the field type "Textarea with variables"? Should actually work (we have clients with Pimcore 6.9 using this). What error do you get?

**Comment by betterapp** [comment]
I tried to do displaying quality progress with information about what fields have error.
In the tutorial Calculated Value have HTML type which does not exist in Pim6.9

![image](https://user-images.githubusercontent.com/7449124/229105566-c0da026f-b3a4-4a32-9924-0f67ccef9055.png)


What I see here is that we have 
![image](https://user-images.githubusercontent.com/7449124/229105663-d10d65d1-a4df-4ec1-9563-dcfeb82fa4a1.png)

And under type there is HTML.
But after setup it does not show anything.

**Comment by BlackbitDevs** [comment]
You are right, type=HTML for calculated value fields only exists from Pimcore 10. Alternatively you can apply https://github.com/pimcore/pimcore/pull/7888/files as a patch (via https://github.com/cweagans/composer-patches/tree/1.x)

But I just took a look at the `Calculated Value (Data Query Selector)` field type. Actually this was thought to be not used for imports but the calculation logic should be written in the configuration field.
But I have now changed it to also support importing into this field type (as long as there is calculation logic entered in the field configuration). Can you please try `3.3.x-dev`? After installing, please save your data object class which contains the data quality field (as I had to change the getter code generation a little bit).
